x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -202,7 +225,14 @@ export const MisconfigurationFindingsDetailsTable = memo(
             <EuiIcon type={'popout'} />
           </SecuritySolutionLinkAnchor>
           <EuiSpacer size="xl" />
-          <DistributionBar stats={getFindingsStats(passedFindings, failedFindings)} />
+          <DistributionBar
+            stats={getFindingsStats(
```

Comment:
:opinion

We should be passing the literal parameters instead of an object.  If I was to look at this in the code I would be expecting only data to passed and not functions like the filter and the reset.

Reply:
wdyt?